charon-ml/src/Meta.ml

```diff
@@ -43,3 +43,17 @@ type meta = {
       (** Where the code actually comes from, in case of macro expansion/inlining/etc. *)
 }
 [@@deriving show, ord]
+
+type inline_attr =
+  | Hint  (** `#[inline]` **)
+  | Never  (** `#[inline(never)]` **)
+  | Always  (** `#[inline(always)]` **)
+[@@deriving show, ord]
+
+type item_meta = {
+  meta : meta;
```

Comment:
Given that there are now several types of meta, could we rename this one in something more explicit? e.g., span_meta? @sonmarcho , wdyt?

Reply:
I agree. Let's do that in a subsequent PR (the diff will be huge). @Nadrieril can you open a PR for this?

Reply:
you mean an issue? I can also open a PR but that'll take a moment ^^